src/affordance/tasks/tools.py

```diff
@@ -25,7 +25,49 @@
 - (str) the output
 - (str) runtime details to be logged
 """
+import pickle
 
+def get_run_dict(run_title):
+    try:
+        with open(f'{run_title}.pickle', 'rb') as handle:
+            run_dict = pickle.load(handle)
+    except FileNotFoundError:
+        run_dict = {}
+        print('run dict not found, creating a new one!')
+    return run_dict
+
+def save_run_dict(run_dict, run_title):
+    with open(f'{run_title}.pickle', 'wb') as handle:
```

Comment:
We should make it possible to choose the directory for this with a flag. Or to save time we should just change it to somewhere not in cwd, maybe `./kvstore` or something?